_articles/Pop!_OS-20.04-LTS-Release-Notes

```diff
@@ -0,0 +1,70 @@
+keywords:
+  - Pop
+  - 20.04
+  - LTS
+  - Updates
+  - New Features
+image: http://support.system76.com/images/system76.png
+hidden: true
+section: pop
+
+---
+
+Pop!\_OS 20.04 was released April 30, 2020.
+
+## Pop!\_OS 20.04 Release Notes
+
+To advance keyboard driven control, keyboard shortcuts have been modified in Pop!\_OS 20.04.
+
+### Notable Shortcut Changes
+
+|          Action            |     Old Shortcut      |      New Shortcut  |
+|:---------------------------| :-------------------- |--------------------|
+Lock Screen                 | <kbd><span class="fl-pop-key"></span></kbd> + <kbd>L</kbd> | <kbd><span class="fl-pop-key"></span></kbd> + <kbd>ESC</kbd> |
+Move to Workspace Up or Down | <kbd><span class="fl-pop-key"></span></kbd> + <kbd>↑</kbd>/<kbd>↓</kbd> | <kbd><span class="fl-pop-key"></span></kbd> + <kbd>Ctrl</kbd> + <kbd>↑</kbd>/<kbd>↓</kbd> |
+Close Window                 | <kbd><span class="fl-pop-key"></span></kbd> + <kbd>W</kbd> | <kbd><span class="fl-pop-key"></span></kbd> + <kbd>Q</kbd>
+Toggle Maximize              | <kbd><span class="fl-pop-key"></span></kbd> + <kbd>↑</kbd> | <kbd><span class="fl-pop-key"></span></kbd> + <kbd>M</kbd>
+
+### New Features
+
+#### Pop Shell Automatic Window Tiling
+Pop!\_OS 20.04 features automatic window tiling. To activate tiling mode, click the tiling menu icon at the top right of the screen and toggle on Tile Windows. Open and newly launched windows will be automatically tiled. Toggling off Tile Windows will revert to floating mode. Click the tiling menu icon again and choose Keyboard Shortcuts. This help application will show you how to navigate, move, and resize windows all with your keyboard.
+
+#### New Application Switcher and Launcher
+Type `Super` + `/` to activate the new application switcher and launcher. Instead of `Super` + `Tabbing` through application icons, simply type `Super` + `/` and start typing the name of the application you want to switch to or launch.
+
+#### Flatpak support with the Flathub
+Pop!\_Shop now includes Flatpak application support and the Flathub application repository by default. The source of applications is visible when viewing applications details. System76 continues to curate and package key applications such as Steam, Atom, and VS Code. Pop!\_OS repositories are prioritized to provide the best user experience for these applications.
+
+#### Hybrid Graphics
+Customers with integrated and discrete graphics can now choose Hybrid Graphics from the battery section of the user menu. While in Hybrid Graphics mode, right clicking an application in the Activities overview offers a "Launch Using Dedicated Graphics Card" option. This enables customers to use battery-saving integrated graphics while launching specific apps on the dedicated GPU.
+
+#### Offline Upgrades
+Version upgrades allow you to download the upgrade in the background and then apply it offline when you’re ready to perform the upgrade.
+
+#### Application Settings in Top Bar
```

Comment:
Le's go with "Application Indicators in Top Bar".

Reply:
ok!